### PR TITLE
Added instructions for releasing a new version

### PR DESCRIPTION
**Pull request by clintonb**

### CODE REVIEW

File: RELEASING.rst
Comment (mwarkentin) [inline RELEASING.rst:26]: Travis can do this bit for you on tags if you’re interested.

**Comment by clintonb** [comment]
I’m aware. I was hesitant because I don’t want to encode my own
creeentials. However, we can create a service user account.

Thanks for the reminder.
-- 
Clinton

**Comment by clintonb** [comment]
@jsocol please review

**Comment by jsocol** [comment]
I have some time set aside this weekend to look at this and #289 / #70 / #181

**Comment by clintonb** [comment]
Ping @jsocol. Any opposition to granting me rights to merge my own PRs?

**Comment by clintonb** [comment]
Ping @jsocol. Any opposition to granting me rights to merge my own PRs?

**Comment by clintonb** [comment]
Paging @jsocol

**Comment by clintonb** [comment]
Ping @jsocol. Any opposition to granting me rights to merge my own PRs?

**Comment by clintonb** [comment]
@jsocol please respond. I gotta admit I'm a bit hurt I haven't received a response after 4.5 months.

**Comment by clintonb** [comment]
@jsocol ???

**Comment by jsocol** [comment]
Sorry all, especially @clintonb. I've been going through some stuff in my personal life. I'm going to have time to come back to waffle in the next couple weeks.

**Comment by clintonb-stripe** [comment]
Thanks!

**Comment by clintonb** [comment]
@jsocol I understand you are going through things in your personal life. Please, let me help by relieving some of the burden here. I am happy to help hold down the fort here, but I _need_ merge rights for my own PRs; otherwise, I can no longer help out here. There are a lot of people who want to see this project be successful, and we are currently doing them a disservice with our inability to review changes in a timely manner or create releases.

I want to help, but I'm limited by my inability to complete a release without your approval.

**Comment by clintonb** [comment]
@jsocol ???

**Comment by clintonb** [comment]
@jsocol please review.

**Review by jsocol — APPROVED** [review]

**Pull request merged**
